### PR TITLE
Mark cloud as beta

### PR DESCRIPTION
**Pull request by MrFlashAccount**
### Pull Request Description

Closes: enso-org/cloud-v2#1466

This PR:
1. Makes `local` as default choice in the sidebar (if appropriate)
2. Add new `<Badge />` component
3. Add `Beta` badge on Cloud item in the Sidebar

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.

### CODE REVIEW

File: app/dashboard/src/components/AriaComponents/Text/Text.tsx [hunk 1]
```diff
@@ -48,8 +48,12 @@ export const TEXT_STYLE = twv.tv({
     variant: {
       custom: '',
       body: 'text-xs leading-[20px] before:h-[1px] after:h-[3px] font-medium',
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      'body-sm': 'text-[10.5px] leading-[16px] before:h-[0.5px] after:h-[2.5px] font-medium',
```
Comment (somebody1234) [inline app/dashboard/src/components/AriaComponents/Text/Text.tsx:52]: i guess personally i would prefer adding new text size(s) for consistency. same applies to the new smaller font size below

see: <https://github.com/enso-org/enso/pull/10924/files?w=1#diff-e2514dc865111604d2abd920738e7f4f3827387245dae778d88e378590288c65R68>

**Review by somebody1234 — APPROVED** [review]
CR :white_check_mark:

**Comment by PabloBuchu** [comment]
Can't build
```
 INFO ide_ci::program::command: pnpm ℹ️ src/components/Devtools/EnsoDevtools.tsx(166,51): error TS17001: JSX elements cannot have multiple attributes with the same name.
 INFO ide_ci::program::command: pnpm ℹ️ /Users/pawelbuchowski/projects/enso/app/dashboard:
 INFO ide_ci::program::command: pnpm ℹ️  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  enso-dashboard@0.1.0 compile: `tsc`
 INFO ide_ci::program::command: pnpm ℹ️ Exit status 2
 INFO ide_ci::program::command: pnpm ℹ️ /Users/pawelbuchowski/projects/enso/app/gui2:
 INFO ide_ci::program::command: pnpm ℹ️  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  enso-gui2@0.1.0 build: `corepack pnpm -r --filter enso-dashboard run compile && corepack pnpm run build:vite`
```

**Comment by MrFlashAccount** [comment]
@PabloBuchu fixed

**Review by PabloBuchu — APPROVED** [review]

**Pull request merged**